tests/jballerina-integration-test/src/test/resources/identifier/PackageNameClashProject/main.bal

```diff
@@ -14,6 +14,10 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import a_b/foo;
+import ballerina/test;
```

Comment:
Previously we had imported from the same project module and different project modules. Why we have change the example?

import a_b/foo;
import a/main.b_foo;

Reply:
This test was disabled before the slp5 Project API changes, where we didn't have a root module concept.
Therefore it was checked using two different imported modules that are used in another module (`main`)
`a/b_foo` and `a_b/foo`
If we have a non-root module with name `b_foo`, it will always be `a/main.b_foo` and not cause a clash.
Therefore, I changed the root module name to `b_foo`

Added another non-root module with the same name in latest commit